_posts/journaling-as-a-continuous-learning-process.md

```diff
@@ -0,0 +1,45 @@
+
+(emphasis is mine)
+
+I started doing it 20 days ago. It's straightforward; it takes 2 minutes. I'm not listing all the things that I did. As suggested, I'm merely listing things that led to a positive or negative outcome. I'm skipping everything that went as expected. For example, if I planned to sit down and write some code, and I happened to do it, I don't list that in the journal. Instead, if I joined a one on one that led to an unexpected outcome, either positive or negative, I try to analyze the motivations and note them down in that day's journal.
+
+It surprised me that writing down things makes them vividly stick in my head: I don't need to go back and read what I journaled two days ago. While I'm writing, I can see emerging patterns; it's clear that certain behaviors, actions, or environments lead to positive or negative outcomes. Those are opportunities to improve.
+
+For example, as I said in a previous post, I try to read a lot. By a lot, I mean that I read 40 to 50 articles a week for personal leisure on average. Simultaneously, two books, one work-related and one for entertainment, such as a novel or about some topic I'm interested in, be it economics or psychology. Those habits have a nasty side effect; they put a lot of psychological pressure on me to get things done every day. I feel pressed to read articles and to read books. Previous to journaling, I read as many articles as I could the first thing in the morning. It was giving me a sense of relief: I did it! However, I was reading books only every other day, too little to be satisfying. I noticed that by journaling. I was consistently logging a negative outcome and a positive one related to articles. I swapped the two activities. I read books in the early morning, more or less for half an hour. And use articles as the filler activity throughout the day. The result is that I read the same amount of articles and I also read books every day.
+
+## Conclusion
+
+There are two things that I don't like. Getting at my desk every morning and being faced with the question, "what am I going to do today?," is one. It is an open door for context switching and unproductivity. It can only destroy my day. The second is that I'm bad at is brainstorming alone. If I need to sit down and think about questions like "what can I improve?" or "what's going wrong?" it'll get me nowhere. Carefully planning my week proved to be extremely effective in answering the first question. Journaling appears to be influential in shaping what's next.
```

Comment:
```suggestion
There are two things that I don't like. Getting at my desk every morning and being faced with the question, "what am I going to do today?," is one. It is an open door for context switching and non-productivity. It can only destroy my day. The second is that I'm bad at brainstorming alone. If I need to sit down and think about questions like "what can I improve?" or "what's going wrong?" it'll get me nowhere. Carefully planning my week proved to be extremely effective in answering the first question. Journaling appears to be influential in shaping what's next.
```